src/react-themer/index.js

```diff
@@ -44,6 +45,12 @@ export default (customThemer: ?Object) => (theme?: Object) => (component: React.
     componentWillMount() {
       const { theme: globalTheme } = this.context;
 
+      Main.PropTypes = {};
```

Comment:
@bevanhunt it looks like Travis is breaking when it runs `npm run flow` for this code: 
![image](https://cloud.githubusercontent.com/assets/1179291/22666376/2a83516e-ec6d-11e6-988f-06d1aa80d4fe.png)